packages/js/plugins/uri-resolvers/ipfs-resolver/src/index.ts

```diff
@@ -40,7 +40,8 @@ export class IpfsResolverPlugin extends Module<NoConfig> {
         {
           cid: `${args.path}/${manifestSearchPattern}`,
           options: {
-            timeout: 5000,
```

Comment:
By removing the default timeout values, it looks like the timeouts will default to 0. I don't know much about IPFS timeouts. Is 0 unlimited timeout? Is this a good default value? Or should we keep default timeout values somewhere? For example, we could have default timeouts in the ipfs uri resolver like `this.env.timeouts?.tryResolveUri ?? 5000`, or we could add a default `env` in the default config. What do you think?

Reply:
Yes, a zero is an unlimited timeout. We could go with something simple like a single `defaultTimeout` which we could use for all actions? Seems like an okay solution, and we don't necessarily need to have a full default local `env` (especially since we wouldn't be defining all `env` defaults, only certain fields).

Reply:
That makes sense to me. I don't know the best way to handle this. I just wanted to bring up the question. I think the default behavior should be that it "just works" without having to configure timeouts, and timeout configuration should be optional. If unlimited timeouts won't cause any issues (like an app hanging, waiting for a response that will never come) then that would be fine.